src/lotto_game/LottoManager.java

```diff
@@ -0,0 +1,35 @@
+package lotto_game;
+
+import lotto_game.domain.LottoGames;
+import lotto_game.view.InputView;
+import lotto_game.view.OutputView;
+
+public class LottoManager {
+    public static final int EACH_LOTTO_PRICE = 1000;
+
+    public void startLottoGames() {
+        InputView inputView = new InputView();
+        OutputView outputView = new OutputView();
+
+        int purchaseMoney = inputView.inputPrice();
+        int numberOfLottoGames = purchaseMoney / EACH_LOTTO_PRICE;
+
+        // 로또 게임 생성
+        LottoGames lottoGames = new LottoGames();
+        lottoGames.initLottoGames(numberOfLottoGames);
```

Comment:
```suggestion
        LottoGames lottoGames = new LottoGames(s(numberOfLottoGames);
```

이렇게 하는 거랑 어떤 차이가 있는지 잘 모르겠어요 ㅎㅎ.  
다양한 학습자료에서 제안하는 아이디어가 언제나 100% 적용해서 좋은 결과를 주는 것은 아닙니다.
왜 사용해야 하는지 스스로 느낄 수 있는 이유를 찾아보고 적용해 보세요.